ertviz/models/realization.py

```diff
@@ -1,18 +1,4 @@
-from ertviz.data_loader import get_data
-
-
 class Realization:
     def __init__(self, realization_schema):
-        self._name = realization_schema["name"]
-        self._data_url = realization_schema["data_url"]
-        self._data = None
-
-    @property
-    def data(self):
-        if self._data is None:
-            self._data = get_data(self._data_url)
-        return self._data
-
-    @property
-    def name(self):
-        return self._name
+        self.name = realization_schema["name"]
+        self.data = realization_schema["data"]
```

Comment:
if the `realization_schema["data"]` can be both an array of data and a url, how will the actual data be retrieved when it is a URL?

Reply:
🤔  you probably have a point, but i don't get it...? can it be both? I think we are only sending "data" now along with the values?

Reply:
in the test data set it seems to be urls

Reply:
hmm.... where exactly? I think i've changed all - but i'm might be mistaken here?

Reply:
forget it - now i see! 👍  will fix!